### PR TITLE
feat: Focus on first focusable element when opening right menu (ACC-296)

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-296" title="ACC-296" target="_blank">SubtaskACC-296</a>  Focus switch on opening User Profile/Group Information
  </td></table>
  <br />
 

Focus on first focusable element when opening right menu (ACC-296)

### CODE REVIEW

File: src/script/page/RightSidebar/RightSidebar.tsx [hunk 1]
```diff
@@ -166,6 +166,18 @@ const RightSidebar: FC<RightSidebarProps> = ({
     });
   }, []);
 
+  const containerRef = useCallback(
+    (element: HTMLDivElement | null) => {
+      const focussableElements =
+        'a:not([disabled]), button:not([disabled]), input[type=text]:not([disabled]), [tabindex]:not([disabled]):not([tabindex="-1"])';
```
Comment (PatrykBuniX) [inline src/script/page/RightSidebar/RightSidebar.tsx]: Maybe we should move this string to some const? We're using it in two places now (check out `util.ts` file)

Comment (PatrykBuniX) [reply src/script/page/RightSidebar/RightSidebar.tsx]: + maybe also create a custom hook for this functionality e.g `useFocusFirstElement` that will return ref callback 🤔, wdyt? seems pretty reusable to me

File: src/script/page/RightSidebar/RightSidebar.tsx [hunk 2]
```diff
@@ -166,6 +166,18 @@ const RightSidebar: FC<RightSidebarProps> = ({
...
+      const nextElementToFocus = element?.querySelectorAll(focussableElements)[0] as HTMLElement;
```
Comment (V-Gira) [inline src/script/page/RightSidebar/RightSidebar.tsx]: Damn, would have taken forever to figure out :D :+1:

Comment (atomrc) [inline src/script/page/RightSidebar/RightSidebar.tsx]:
```suggestion
      const nextElementToFocus = element?.querySelector(focussableElements) as HTMLElement;
```

Comment (thisisamir98) [reply src/script/page/RightSidebar/RightSidebar.tsx]: Found it on https://stackoverflow.com/a/35173443/7091035

**Review by PatrykBuniX — APPROVED** [review]

**Pull request merged**